CIP-0120/README.md

```diff
@@ -0,0 +1,411 @@
+replace this one.
+
+### Terminology
+
+#### Capitalisation of key terms
+
+To avoid unnecessary edits, and therefore checksum changes, 
+constitution authors MUST follow the following standard English capitalisation 
+rules unless a translation language indicates otherwise:
+
+`Constitution` vs. `constitution`
+
+- The Constitution in effect — either the "initial" one or any updated version — is unique and therefore capitalised ("Constitution") as a proper noun.
+- Draft or proposed constitutions are not unique & therefore are not capitalised ("constitution") as a common noun.
+- "Cardano Constitution" is a very specific proper noun phrase (also a title) and so each word is capitalised.
+- The phrase "the Constitution", unless used non-specifically (e.g. "the constitution that voters prefer"), would generally be assumed to be _the_ Constitution and capitalised as a proper noun.
+
+`Ada` vs. `ada` vs. `ADA`
+
+- `Ada` is the currency, while `ada` indicates units of that currency (e.g. "Ada holders can accumulate more ada to increase their influence.")
+- `ADA` is the trading symbol (e.g. "Fluctuations in ADA might influence decisions about the Treasury.")
+
+### Characters
+
+The constitution text MUST only contain UTF-8 characters.
+
+### Lines
+
+Each line in the constitution text MUST contain at maximum 80 characters,
+including spaces and punctuation.
+
+#### Sentences
+
+The constitution text MUST only contain a maximum of one sentence per group of 
+lines delimited by one blank line between these groups.
+Each sentence MUST be followed by a newline.
+
+Each sentence SHOULD start on its own line with a capitalized letter.
+
+Example:
+
+```md
+This is a short sentence on one line.
+
+This is a long sentence and I have valid reasons for it being so long,
+such as being an example of a long sentence.
+```
+
+While 80 characters is a limit, authors don't have to try and always hit 80.
+Legibility should be kept in mind.
+
+### File Type
+
+The constitution file MUST be a text file named `constitution.txt`.
+
+### Hashing
+
+When supplying a constitution hash digest to chain, the algorithm used MUST be 
+Blake2b-256.
+Before creating a hash digest the constitution plain text MUST be in its raw 
+text, including any [Rich Text Formatting](#rich-text-formatting) 
+related characters.
+
+### Storage
+
+The each ratified constitution MUST be stored, 
+immutably on a distributed storage mechanism such as IPFS.
+Where backups can be easily made in a permissionless manner by 
+interested parties.
+This storage platform MUST be easily accessible, with strong tooling support.
+Authors SHOULD NOT specify the use of centralized gateways to access 
+the constitution plain text.
+
+### Rich Text Formatting
+
+The constitution text MAY include a strict subset of rich text styling as 
+defined in this specification.
+Tooling rendering the constitution SHOULD recognize these and render them 
+faithfully.
+
+#### Headers
```

Comment:
Should we specify that each header and list should have a blank line before and after it?  It seems like a common markdown error that we could easily avoid.

Reply:
Yup!
I will add this as a SHOULD

Reply:
addressed in  [1b87d84](https://github.com/cardano-foundation/CIPs/pull/796/commits/1b87d8463505e4d5943e2226a0894c20f45bcbb3)